BuildingSync.xsd

```diff
@@ -4744,9 +4784,9 @@
 											</xs:sequence>
 										</xs:complexType>
 									</xs:element>
-									<xs:element name="ElectricMotorsAndDrives" minOccurs="0">
+									<xs:element name="OtherElectricMotorsAndDrives" minOccurs="0">
```

Comment:
This is the change that caught out the example files.

Reply:
@jasondegraw  thanks for finding this.  I had forgotten that this involved me changing an existing BSync category name.  The reason is that I wanted to differentiate this from Conveyance Systems, which also include large electric motors and drives (i.e., not plug loads).  We could say "NonConveyance" instead of "Other" in that name.  Regardless, it would be a name change.  The only other change to existing verbage in my PR was in one of the enums under the "ChilledWaterHotWaterAndSteamDistributionSystems" category.  I changed DHW to SHW.  Let me know if you think any of these is a problem.